src/corecel/sys/Stream.cc

```diff
@@ -209,8 +208,7 @@ Stream::Stream(Stream&& other) noexcept
  */
 Stream& Stream::operator=(Stream&& other) noexcept
 {
-    Stream temp(std::move(other));
-    this->swap(temp);
+    this->swap(other);
```

Comment:
Right, blank state is better. This didn't trigger a warning; the other constructor did, and I noticed that extra temporary in the move assignment. I figured since move semantics are supposed to be an optimization of copy, I wanted to remove it but the compiler might be able to do it, I'll revert this.